docs/develop/standardsguidelines.md

```
@@ -0,0 +1,93 @@
+## Source Code from an AI agent or bot
+> [!IMPORTANT]
+> Its OK if you took help from an AI for writing your source code. 
+>
+> However, we expect a few things from you as the person making a contribution to MoonLight:
+* Make sure you really understand the code suggested by the AI, and don't just accept it because it "seems to work".
+* Don't let the AI change existing code without double-checking by you as the contributor. Often, the result will not be complete. For example, previous source code comments may be lost.
+* Remember that AI are still "Often-Wrong" ;-)
+* If you don't feel very confident using English, you can use AI for translating code comments and descriptions into English. AI bots are very good at understanding language. However, always check if the results is correct. The translation might still have wrong technical terms, or errors in some details.
+
+### best practice with AI:
+  * As the person who contributes source code to MoonLight, make sure you understand exactly what the AI-generated code does
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)